backend/azure-functions/src/ComingFeedback/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as FacebookAuth from '../Auth/facebook-auth';
+import { getComingFeedbackContainer } from '../Containers/containers';
+import { SingleComingFeedback } from '../Containers/single-coming-feedback';
+import { Context, HttpRequest } from 'azure-functions-ts-essentials';
+import { Container } from '@azure/cosmos';
+import * as knex from 'knex';
+
+const httpTrigger = async function (context: Context, req: HttpRequest): Promise<void> {
+    const reportId: string = req.query.reportId;
+    if (!reportId || reportId.length === 0) {
+        context.res = {
+            status: 400,
+            body: 'Bad request: missing reportId query parameter',
+        };
+
+        context.done();
+        return;
+    }
+
+    const authResult: FacebookAuth.AuthenticationResult = await FacebookAuth.authenticate(req, context);
```

Comment:
Can you set this up as a middleware?
It's supposed to be super simple with that: https://github.com/emanuelcasco/azure-middleware
This way you decouple the logic from the auth and can reuse the auth code everywhere (although we said we want to use the API mngmnt).

Reply:
Also, authentication should be first.

Reply:
Moved auth to be first.
Regards middleware - not part of this PR, it's infra auth change that might also conflict other PRs (that's why I splitted the auth changes for other PRs).